docs/docs/prompt-management/03-prompt-management-sdk.mdx

```diff
@@ -484,7 +533,7 @@ If your application uses asynchronous programming, you can use the async version
 
 ```python
 # Asynchronous fetching of configuration
-config = await ag.ConfigManager.aget_from_registry(
+config = await ag.ConfigManager.async_get_from_registry(
```

Comment:
Why is every async function called `afunc` while `get_from_registry` is called `aget_from_registry`?

Reply:
yes

Reply:
Got it.